apps/coscrad-frontend/src/components/higher-order-components/categorizable-page-layout.tsx

```diff
@@ -72,6 +74,26 @@ export const CategorizablePageLayout = ({
         openPanelLocation: null,
     });
 
+    /**
+     * This is a first step to closing the panel when a connected resource in the open panel has been
+     * clicked.
+     */
+    const [focusResourceId, setFocusResourceId] = useState(id);
```

Comment:
It's not clear to me what's going on here. Why are there 2 competing IDs? The categorizable page should always have an ID in the location and that should flow through the props here.